src/components/activity/ActivityAnchorForm.svelte

```diff
@@ -2,6 +2,8 @@
 
 <script lang="ts">
   import { createEventDispatcher } from 'svelte';
+  import { PlanStatusMessages } from '../../enums/planStatusMessages';
+  import { planReadOnly } from '../../stores/plan';
```

Comment:
The unit test is failing here because the test doesn't have a mocked store. I think either we make this a prop into the component or fix the test to take a mocked store.